src/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py

```diff
@@ -1126,6 +1192,9 @@ def main():
     if options["footprints"]:
         downloader.save_footprints(options["footprints"])
 
+    if flags["s"]:
+        downloader.skip_existing(options["output"], "*.zip")
```

Comment:
When downloading from GCS or USGS_EE, the result is in .SAFE format (not sure about ESA result at the moment). It would be good to scan for this file ending too

Reply:
Right. Some users may also extract the zip, link the SAFE format (`r.external`) and then remove the zip...
Integrated in #609